package.json

```diff
@@ -8,14 +8,14 @@
     "@types/find-config": "^1.0.4",
     "@types/koa-cors": "^0.0.6",
     "@types/sinon": "^10.0.20",
-    "@types/lodash": "^4.17.7",
```

Comment:
Why is this removed? I've seen lodash was used somewhere in the codebase I'd suggest revert this back. This PR also has a different purpose so removing this dep can confuse devs in the future.

Reply:
Restoring the package.json from main